client/me/help/help-contact/index.jsx

```diff
@@ -286,7 +295,11 @@ module.exports = React.createClass( {
 		//Autofill the subject field since we will be showing it now that operators have went away.
 		this.autofillSubject();
 
-		this.showAvailabilityNotice( IS_UNAVAILABLE );
+		if ( timedOut ) {
```

Comment:
@jordwest I've updated it in f20d30d. Do we have steps for testing the operator away case?